config/sync/core.entity_view_display.node.application.default.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   module:
     - link
     - metatag
+    - smart_trim
```

Comment:
Is that expected/desired? Looks out of scope of the ticket but could well be what's needed, regardless.

Reply:
Good question, have investigated and this just seems to be an oddity.
If you pull latest development, import config, make any minor change to the Article display and then export, smart_trim appears ! 
I think the question is not "why is it being added " but is really "why wasn't it there already" - as smart_trim is already being used in Article display.